File: Euler1.java

 /**
  * This program finds the sum of all multiple of 3 or 5
  * below 1000 and outputs them to standard output
  */

  public class Euler1 {

  	 public static void main(String[] args) {

      int sum = 0;

    
      for(int i = 0; i <= 1000; i++) {

         if( i % 3 == 0 || i % 5 == 0)
            sum += i;      

      }

      System.out.println(sum);

    }  

  }

File: euler_1.py
# sum of multiples of 3 or 5 below 1000
max_value = 1000
x = 1
sum = 0
while x < 1000:
   if x % 3 == 0 or x % 5 == 0:
       sum += x
   x += 1

print sum   